runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerOverlay.java

```diff
@@ -57,7 +60,7 @@ private GroundMarkerOverlay(Client client, GroundMarkerConfig config, GroundMark
 		this.plugin = plugin;
 		setPosition(OverlayPosition.DYNAMIC);
 		setPriority(OverlayPriority.LOW);
-		setLayer(OverlayLayer.ABOVE_SCENE);
+		setLayer(OverlayLayer.ABOVE_WIDGETS);
```

Comment:
This will make the current Ground Markers render above elements like the chatbox, which does not look good.

Reply:
it wont render on minimap without that option. Guess i could move to code to a separate overlay class to fix that